addons/base_vat/views/res_partner_views.xml

```diff
@@ -6,9 +6,16 @@
             <field name="name">view.partner.base.vat.form</field>
             <field name="inherit_id" ref="base.view_partner_form" />
             <field name="arch" type="xml">
-                <xpath expr="//div[hasclass('alert')]" position="after">
-                    <div class="alert alert-warning oe_edit_only" role="alert" attrs="{'invisible': [('vies_failed_message', '=', False)]}">
-                        <field name="vies_failed_message" nolabel="1" readonly="1"/>
+                <xpath expr="//field[@name='vat']" position="replace">
+                    <field name="perform_vies_check" invisible="1"/>
+                    <label for="vat" string="Tax ID"/>
+                    <div attrs="{'invisible': [('perform_vies_check', '=', False)]}">
+                        <field name="vat" placeholder="e.g. BE0477472701" attrs="{'readonly': [('parent_id','!=',False)]}" class="oe_inline"/>
+                        <label for="vies_valid"/>
+                        <field name="vies_valid" />
+                    </div>
+                    <div attrs="{'invisible': [('perform_vies_check', '=', True)]}">
+                        <field name="vat" placeholder="e.g. BE0477472701" attrs="{'readonly': [('parent_id','!=', False)]}"/>
```

Comment:
Didn't test, but isn't this more readable/logical?
```suggestion
                    <div>
                        <field name="vat" placeholder="e.g. BE0477472701" attrs="{'readonly': [('parent_id','!=',False)]}" class="oe_inline"/>
                        <label for="vies_valid" attrs="{'invisible': [('perform_vies_check', '=', True)]}"/>
                        <field name="vies_valid" attrs="{'invisible': [('perform_vies_check', '=', True)]}"/>
```

Reply:
This is the difference, it's only cosmetic really, but I wanted to keep the VAT field as close in appearance to how it is normally when the vies checkbox is not visible. 
With the above code:
![image](https://user-images.githubusercontent.com/46533567/234717240-cf9a642a-d27e-4460-bef4-308319696d79.png)

With my code:
![image](https://user-images.githubusercontent.com/46533567/234717275-c132fb79-7100-4d66-89c8-39009d131406.png)